config_test.go

```diff
@@ -102,7 +144,6 @@ receivers:
         - job_name: 'oteltui_prom_2'
           scrape_interval: 5s
           metrics_path: '/custom/prometheus'
-          scheme: 'http'
```

Comment:
fyi I took out scheme as it is default to http. I looked around for https examples and am not aware of a public endpoint you can scrape, which means you have additional config needed to either trust or supply alternative certs. Since we don't have the ability to add such config, nor have been asked for it, I just didn't put in an https example rather than forge one.